app/js/components/labOrderEntry/DraftOrder/DraftList.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import shortid from 'shortid';
+import PropTypes from 'prop-types';
+
+export class DraftList extends React.Component {
+  render() {
+    return this.props.draftOrders.map(order => (
```

Comment:
I don't think there will be any form of component reuse or state management here so you could move it back to a handler where you're exporting it

Reply:
I will need better understanding of what you are trying to say.